packages/invoice-dashboard/src/lib/dashboard/invoice-view.svelte

```diff
@@ -279,12 +281,12 @@
           {#each firstItems as item, index (index)}
             <tr class="table-row item-row">
               <th scope="row" class="item-description">
-                <p class="truncate description-text">{item.name}</p>
+                <p class="truncate description-text">{item.name || "-"}</p>
               </th>
-              <td>{item.quantity}</td>
+              <td>{item.quantity || "-"}</td>
               <td>{formatUnits(item.unitPrice, currency?.decimals ?? 18)}</td>
               <td>{formatUnits(item.discount, currency?.decimals ?? 18)}</td>
-              <td>{Number(item.tax.amount)}</td>
+              <td>{Number(item.tax.amount || "-")}</td>
```

Comment:
We should also make sure to add here and in the discount and such, these might now be available in some requests.